docs/development.md

```diff
@@ -313,7 +313,17 @@ kubebuilder create config \
 ```
 
 Once the installation YAML config `hack/install.yaml` is created, we need to
-delete the `type` field from the OpenAPI schema to avoid triggering validation
+update it to modify a few fields.
```

Comment:
+1 to have a chart here, but currently, the `hack/install.yaml` was generated by `kubebuilder`, so shall we keep both chart and this YAML template? 
1) Using `kubebuilder` to generate this YAML template.
2) Update the helm chart to reflect the latest change generated by `kubebuilder`.

Reply:
We do have a static yaml for deploying the latest version of fedv2 (`hack/install-latest.yaml`) - this is just for the dev workflow which relies on the generated `hack/install.yaml` produced by kubebuilder.

We could do a chart. However, the amount of custom edits will be reduced as we stabilize on a version of kubebuilder and kube that work together. Also, there aren't really many configurable parameters and we rely on kubebuilder to generate the config so it would be mean maintaining another deployment config. I think kubebuilder could be enhanced to allow this to be configurable. I've filed https://github.com/kubernetes-sigs/kubebuilder/issues/356 to track this. Plus, the deployment of fed-v2 is mostly automated already so I'm not sure we'd gain much from a chart at this point.

With that said, I'm open to alternative views.

Reply:
How about have a simple chart and we can iterate to improve it based on fed v2 development.

@font I found kubernetes-sigs/kubebuilder#356 was closed and they recommended to use `kustomize`, but introducing `kustomize` will increase the dependency of fed v2 to other components.

Reply:
@gyliu513 Kubebuilder `v1` already has a dependency on `kustomize`, so it seems unavoidable.

Reply:
Thanks @font , then is it possible that we use `kustomize` to update cpu and memory for fed v2 sts?